app/models/order_detail.rb

```diff
@@ -38,7 +38,7 @@ def set_problem_order
   belongs_to :price_policy
   belongs_to :statement, inverse_of: :order_details
   belongs_to :journal
-  belongs_to :order
+  belongs_to :order, inverse_of: :order_details
```

Comment:
That's where it's at now. It's not obvious to me why the `inverse_of: :order` breaks child order_detail updates. I was hoping it was something in the way the specs are set up, but it doesn't seem to be. They're either not happening or getting reverted.